### PR TITLE
Fix typos, config files

### PR DESCRIPTION
**Pull request by ink8bit**
# Just cleanup
- Fix some typos
- Fix configuration files

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,5 +1,5 @@
 node_modules
 .DS_Store
-npm-debug.log*
-yarn-error.log
+*.log*
 docs/_site
+.idea/
```
Comment (ink8bit) [inline .gitignore]: for those who use JetBrains products

File: src/siema.js [hunk 2]
```diff
@@ -439,7 +439,7 @@ export default class Siema {
     }
```
Comment (ink8bit) [inline src/siema.js]: It seems that you don't use `remove` method anywhere

File: src/siema.js [hunk 3]
```diff
@@ -45,11 +45,8 @@ export default class Siema {
       onInit: () => {},
       onChange: () => {},
     };
-    const userSttings = options;
-    for (const attrname in userSttings) {
-      settings[attrname] = userSttings[attrname];
-    }
-    return settings;
+
+    return Object.assign(settings, options);
```
Comment (ink8bit) [inline src/siema.js]: It's better to use `Object.assign` but you'll have to deal with lack of browser support https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/assign.
If is not possible I can revert this change.

**Comment by pawelgrzybek** [comment]
Hi.

You are a top contributor! Please revert these two minor things that I commented out above and I'm happy to merge. Thanks for your english skills and lots of love that you put to my project. I owe you a good beer @ink8bit

**Comment by ink8bit** [comment]
Hi, @pawelgrzybek!
Thank you! You created a very good project.

You mentioned about two minor fixes I have to revert. But I don't see any comments in my pull request.
Could you please do it again?

**Comment by pawelgrzybek** [comment]
You can have a look above at the review tab. It is basically removing `Object.assign()` from the config and removing check for `this.config.perPage.hasOwnProperty(viewport)`.

**Comment by ink8bit** [comment]
Ok! I reverted those changes.

**Comment by pawelgrzybek** [comment]
Thanks a ton for top contribution!

**Comment by ink8bit** [comment]
You're welcome!

**Pull request merged**